Remove manager and chassis endpoints from Redfish API

amarnath-ac opened the pull request:
- Removed manager and chassis endpoints from OpenAPI specification
- Updated generated server code and types accordingly

amarnath-ac committed:
refactor: remove manager and chassis endpoints from Redfish API

- Removed manager and chassis endpoints from OpenAPI specification
- Updated generated server code and types accordingly

Signed-off-by: C, Amarnath <amarnath.c@intel.com>

sudhir-intc reviewed (commented):
current changes LGTM you need to remove the following files too from redfish/openapi/dmtf:
Chassis.v1_28_0.yaml
ChassisCollection.yaml
Manager.v1_23_0.yaml
ManagerCollection.yaml
Also check if there are any other references for Chassis and Managers in the other DMTF spec files

amarnath-ac commented:
> current changes LGTM you need to remove the following files too from redfish/openapi/dmtf: Chassis.v1_28_0.yaml ChassisCollection.yaml Manager.v1_23_0.yaml ManagerCollection.yaml Also check if there are any other references for Chassis and Managers in the other DMTF spec files

Fixed

amarnath-ac committed:
refactor: remove chassis and manager spec files

Remove chassis and manager related OpenAPI specifications and update route handlers to exclude these endpoints from the service root response.

sudhir-intc reviewed (approved):
LGTM

sudhir-intc committed:
Merge branch 'redfish-plugin-arch' into remove_spec_manager_chassis
